libsolidity/CMakeLists.txt

```diff
@@ -166,7 +166,7 @@ if (NOT (${Z3_FOUND} OR ${CVC4_FOUND}))
 endif()
 
 add_library(solidity ${sources} ${z3_SRCS} ${cvc4_SRCS})
-target_link_libraries(solidity PUBLIC yul evmasm langutil solutil Boost::boost Boost::filesystem Boost::system)
+target_link_libraries(solidity PUBLIC yul evmasm langutil solutil Boost::boost)
```

Comment:
In that sense we don't even need ``Boost::boost`` here, since ``solutil`` is publicly linked against it already.

Reply:
That's true, but perhaps `libsolidity` uses some boost features in its own right and it makes sense to not implicitly include it via `libsolutil` (it does not make any difference though).